<launch>
  <arg name="arm_left"  default="True"/>
  <arg name="arm_right"  default="True"/>
  <arg name="end_effector_left"  default="pal-gripper"/>
  <arg name="end_effector_right"  default="pal-gripper"/>
  <arg name="ft_sensor_left"  default="schunk-ft"/>
  <arg name="ft_sensor_right"  default="schunk-ft"/>
  <arg name="laser_model"  default="sick-571"/>
  <arg name="camera_model"  default="orbbec-astra"/>
  <arg name="has_screen"  default="False"/>
  <arg name="base_type"  default="pmb2"/>
  <!-- Args are the ones that upload.launch requires-->
  <include file="$(find tiago_dual_description)/robots/upload.launch">
    <arg name="arm_left" value="$(arg arm_left)"/>
    <arg name="arm_right" value="$(arg arm_right)"/>
    <arg name="end_effector_left"  value="$(arg end_effector_left)"/>
    <arg name="end_effector_right"  value="$(arg end_effector_right)"/>
    <arg name="ft_sensor_left"  value="$(arg ft_sensor_left)"/>
    <arg name="ft_sensor_right"  value="$(arg ft_sensor_right)"/>
    <arg name="laser_model"  value="$(arg laser_model)"/>
    <arg name="camera_model"  value="$(arg camera_model)"/>
    <arg name="has_screen"  value="$(arg has_screen)"/>
    <arg name="base_type"  value="$(arg base_type)"/>
  </include>

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
  <!--node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" args="_use_gui:=True" /-->

  <!--node name="rviz" pkg="rviz" type="rviz"/-->
</launch>
